languages/gutenberg.pot

```diff
@@ -15,22 +15,26 @@ msgstr ""
 msgid "Image"
 msgstr ""
 
-#: editor/blocks/list/index.js:5
-msgid "List"
-msgstr ""
-
-#: editor/blocks/text/index.js:19
+#: editor/blocks/list/index.js:22
 msgid "Align left"
 msgstr ""
 
-#: editor/blocks/text/index.js:27
+#: editor/blocks/list/index.js:30
 msgid "Align center"
 msgstr ""
 
-#: editor/blocks/text/index.js:35
+#: editor/blocks/list/index.js:38
```

Comment:
Ah, based on these changes, I think we'll want to tune the string extraction to do a better job of identifying strings that are present in multiple files.